docs/source/tools.rst

```diff
@@ -0,0 +1,79 @@
+
+Tools Guide
+=================
+
+First, make sure that PyU4V is `installed <http://pyu4v.readthedocs.io/en/latest/installation.html>`_.
+Then visit the `quick_start <http://pyu4v.readthedocs.io/en/latest/quick_start.html>`_ section to make
```

Comment:
Lines run over 79 chars, you can shorten links by adding them to the bottom of the file in the same way as the other files